src/components/settings/FilterSettings.svelte

```diff
@@ -63,6 +64,12 @@
     <CheckOption name="MChad (volunteer translators)" store={enableMchadTLs} />
   </Col>
 </Row>
+{#if $enableMchadTLs}
+  <MultiDropdown
+    name="Blocked users (MChad)"
+    store={mchadUsers}
+  />
+{/if}
```

Comment:
May I suggest putting both "blocked users" dropdowns next to each other? Either moving the YT one down or the MChad one up.

Reply:
Done

![image](https://user-images.githubusercontent.com/50760816/127810887-42da1e3a-9db2-4104-8acc-8c8914d3dac5.png)